actors/miner/src/lib.rs

```diff
@@ -142,17 +142,19 @@ impl Actor {
             .map(|address| resolve_control_address(rt, address))
             .collect::<Result<_, _>>()?;
 
+        let policy = rt.policy();
```

Comment:
nit: Might as well do this above 133 so everyone can use the local variable to cut down on three function calls

Reply:
you would think!!!

thats what i had in the beginning, but then i met the borrow checker and he screamed at me...